src/pages/IndexPage.vue

```
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { useMessagesStore } from "@/stores/messages";
+import MessageItem from "~/layout/MessageItem.vue";
+import SVGButton from "@/components/ui/button/SVGButton.vue";
+
+const { messages } = storeToRefs(useMessagesStore());
+const { fetchMessages } = useMessagesStore();
+
+const loading = ref<boolean>(false);
+async function refetch() {
```

ESLint

[error] 'refetch' is defined but never used.

(@typescript-eslint/no-unused-vars)